doc/content/source/kernels/MFEMVectorFEDomainLFKernel.md

```diff
@@ -9,7 +9,7 @@ Adds the domain integrator for integrating the linear form
 !equation
 (\vec f, \vec v)_\Omega \,\,\, \forall \vec v \in V
 
-where $\vec v \in H(\mathrm{curl})$ or $H(\mathrm{div})$ is the trial variable and $\vec f$ is a
+where $\vec v \in H(\mathrm{curl})$ or $H(\mathrm{div})$ is the test variable and $\vec f$ is a
```

Comment:
So if I understand correctly, the only difference between `MFEMVectorDomainLFKernel` and `MFEMVectorFEDomainLFKernel` is that the former is for H1 test variables and the latter for HDiv and HCurl ones. I'm not sure the class names reflect this fact, at first glance it's not obvious to me from the names what the difference is.

Reply:
Indeed - this is the difference. I've erred for consistency with MFEM here, but I do agree that this isn't the most user-friendly naming convention for this reason. May be worth having similar kernels merged to avoid this long-term, but I'd be keen to keep that for a separate PR/discussion